.gitignore

```diff
@@ -1,2 +1,5 @@
 # IDE-specific metadata
 .idea/
+
+# Environment variables. Useful for examples.
+.env
```

Comment:
Creating a new thread as this trailing newline has not been fixed yet.

Reply:
It's fixed, but I just hasn't pushed it yet.